index.html

```diff
@@ -5,8 +5,97 @@
+          <li><a href="blank.html">Гостиницы</a></li>
+        </Ul>
+      </nav>
+    </header>
+    <main>
+      <a>
+        <img src="https://via.placeholder.com/150x100" alt="welcome">
+        <img src="https://via.placeholder.com/150x100" alt="tothegorgeous">
+        <img src="https://via.placeholder.com/150x100" alt="SEDONA">
+      </a>
+      <h2>Седона — небольшой городок в Аризоне, заслуживающий большего! </h2>
+      <p>Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</p>
+    <section>
+      <h2>Настоящий городок</h2>
+      <p>— №1 —</p>
+      <p>Седона — не аттракцион для туристов, там течёт своя жизнь </p>
+      <img src="https://via.placeholder.com/150x100" alt="city">
+      <ul>
+         <li>
+          <img src="https://via.placeholder.com/150x100" alt="house"> 
+          <h2>Жильё</h2>
+           <p>Жильё  Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+         </li>
+         <li>
+            <img src="https://via.placeholder.com/150x100" alt="food">
+            <h2>Еда</h2>
+            <p>Еда  Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+          <li>
+            <img src="https://via.placeholder.com/150x100" alt="souvenir">
+            <h2>Сувениры</h2>
+            <p>Не только китайского, но и местного производства!</p>
+          </li>
+      </ul>
+    </section>
+    <section>
+      <h2>Там есть Мост дьявола</h2>
+      <p>— №2 —</p>
+      <p>Да, по нему можно пройти! Если конечно вы осмелитесь</p>
+      <img src="https://via.placeholder.com/150x100" alt="bridge">
+      <ul>
+         <li>
+          <img src="https://via.placeholder.com/150x100" alt="space"> 
+          <h2>Небольшая
+            площадь</h2>
+            <p>— №3 —</p>
+           <p>Жильё  Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+         </li>
+         <li>
+          <img src="https://via.placeholder.com/150x100" alt="row">  
+          <h2>Красивая дорога</h2>
+            <p>— №4 —</p>
+            <p>Ехать в седону из лас-вегаса совсем не скучно!</p>
+          </li>
+          <li>
+            <img src="https://via.placeholder.com/150x100" alt="tourists">
+            <h2>Мало туристов</h2>
+            <p>— №5 —</p>
+            <p>Большинство едет в гранд каньон и толпится там</p>
+          </li>
+      </ul>
+    </section>
+    <section>
+      <ul>
+        <li>
+          <h1>Заинтересовались?</h1>
+        </li>
+        <li>
+          <p>Укажите предполагаемые даты поездки,  и мы покажем вам лучшие предложения гостиниц в седоне</p>
+        </li>
+        <li>
+          <h2><a href="blank.html">Поиск гостиницы в Седоне</a></h2>
+        </li>
+      </ul>
+      <img src="https://via.placeholder.com/150x100" alt="map">
+    </section>
+    </main>
+    <footer>
+      <ul>
+        <li>
+          <h2>#VISITSEDONA</h2>
+          <h2><a href="tel:+78005558628">+7 800 555-86-28</h2>
```

Comment:
```suggestion
          <h2><a href="tel:+78005558628">+7 800 555-86-28</a></h2>
```
Пропущен закрывающий тег `</a>`.